Shared/Realm.Shared/RealmConfigurationBase.cs

```diff
@@ -35,24 +38,32 @@ public abstract class RealmConfigurationBase
         public static string DefaultRealmName => "default.realm";
 
         /// <summary>
-        /// Gets or sets the full path of the realms opened with this configuration, may be overriden by passing in a separate name.
+        /// Gets or sets the full path of the Realms opened with this Configuration. May be overriden by passing in a separate name.
         /// </summary>
+        /// <value>The absolute path to the Realm.</value>
         public string DatabasePath { get; protected set; }
 
         internal bool Dynamic;
 
         /// <summary>
         /// Gets or sets the list of classes persisted in a Realm opened with this configuration.
         /// </summary>
-        /// <remarks>Specify classes by type. Searched linearly so order in decreasing frequency of creating objects.</remarks>
-        /// <example>For example: `config.ObjectClasses = new Type[] { typeof(CommonClass), typeof(RareClass) };`.</example>
-        /// <value>Typically left null so by default all RealmObjects will be able to be stored in all realms.</value>
+        /// <example>
+        /// <code>
+        /// config.ObjectClasses = new Type[] 
+        /// { 
+        ///     typeof(CommonClass), 
+        ///     typeof(RareClass) 
+        /// };
+        /// </code>
+        /// </example>
+        /// <value>Typically left null so by default all <see cref="RealmObject"/>s will be able to be stored in all Realms.</value>
```

Comment:
This seems inconsistent with what the `<value>` elements normally contain -- this is more like a remark.